src/webgpu/api/validation/validation_test.ts

```diff
@@ -3,6 +3,30 @@ import { BindableResource } from '../../capability_info.js';
 import { GPUTest } from '../../gpu_test.js';
 
 export class ValidationTest extends GPUTest {
+  createTextureWithState(
+    state: 'valid' | 'invalid' | 'destroyed',
+    descriptor: GPUTextureDescriptor
+  ): GPUTexture {
+    let texture: GPUTexture;
+    switch (state) {
+      case 'valid':
+        texture = this.device.createTexture(descriptor);
+        break;
+      case 'invalid':
+        this.device.pushErrorScope('validation');
+        texture = this.device.createTexture(descriptor);
+        this.device.popErrorScope();
```

Comment:
This case should just return getErrorTexture().

Reply:
Done. And change a bit more for this helper function (make descriptor as optional and have a default descriptor in it). Pls take a look at whether this change is Ok, thanks!